x/oracle/keeper/keeper_test.go

```diff
@@ -71,11 +75,17 @@ func (s *IntegrationTestSuite) SetupTest() {
 
 // Test addresses
 var (
-	valPubKeys = simapp.CreateTestPubKeys(1)
-	valPubKey  = valPubKeys[0]
-	pubKey     = secp256k1.GenPrivKey().PubKey()
-	addr       = sdk.AccAddress(pubKey.Address())
-	valAddr    = sdk.ValAddress(pubKey.Address())
+	valPubKeys = simapp.CreateTestPubKeys(2)
+
+	valPubKey = valPubKeys[0]
+	pubKey    = secp256k1.GenPrivKey().PubKey()
+	addr      = sdk.AccAddress(pubKey.Address())
+	valAddr   = sdk.ValAddress(pubKey.Address())
+
+	valPubKey2 = valPubKeys[1]
+	pubKey2    = secp256k1.GenPrivKey().PubKey()
+	addr2      = sdk.AccAddress(pubKey2.Address())
+	valAddr2   = sdk.ValAddress(pubKey2.Address())
```

Comment:
This is kinda ugly, should turn these into arrays in a separate PR just to keep this one clean